py4DSTEM/braggvectors/diskdetection_aiml.py

```diff
@@ -905,7 +907,10 @@ def _get_latest_model(model_path=None):
             with open("./tmp/model_metadata_old.json") as f_old:
                 metaold = json.load(f_old)
                 file_id_old = metaold["file_id"]
-        except:
+        # TODO Double check this is correct Error
+        except FileNotFoundError:
+            file_id_old = file_id
+        except Exception:
```

Comment:
These can be combined if they have the same behavior

Reply:
will combine